### PR TITLE
feat: new error messages

### PR DESCRIPTION
**Pull request by Addminus**

### CODE REVIEW

File: src/execution/exchanges/swap.execute.ts [hunk 1]
```diff
@@ -167,7 +167,7 @@ export class SwapExecutionManager {
         error: {
           code: LifiErrorCode.TransactionFailed,
           message: 'Failed while waiting for receiving chain.',
-          htmlMessage: getTransactionFailedMessage(swapProcess),
+          htmlMessage: getTransactionFailedMessage(step, swapProcess.txHash),
```
Comment (mathiasmoeller) [inline src/execution/exchanges/swap.execute.ts:170]: you are passing the `txHash` here but the method expects a `txLink`. is that on purpose?

File: src/utils/parseError.ts [hunk 2]
```diff
@@ -150,6 +158,14 @@ export const parseError = async (
         e.stack
       )
     }
+
+    if (e.Code === LifiErrorCode.ValidationError) {
```
Comment (mathiasmoeller) [inline src/utils/parseError.ts:162]: Not sure if it makes a difference but it should probably be `e.code`

File: src/execution/exchanges/swap.execute.ts (hunk 1 above)
Comment (Addminus) [reply src/execution/exchanges/swap.execute.ts:170]: that was a mistake

File: src/utils/parseError.ts (hunk 2 above)
Comment (Addminus) [reply src/utils/parseError.ts:162]: that does make a difference. I just copied the code from the transaction unprepared check. There it is `e.Code`. So we are getting parsed messages for this error now too :)

File: src/execution/bridges/bridge.execute.ts [hunk 3]
```diff
@@ -165,7 +165,10 @@ export class BridgeExecutionManager {
         error: {
           code: LifiErrorCode.TransactionFailed,
           message: 'Failed while waiting for receiving chain.',
-          htmlMessage: getTransactionFailedMessage(crossChainProcess),
+          htmlMessage: getTransactionFailedMessage(
+            step,
+            crossChainProcess.txHash
```
Comment (mathiasmoeller) [inline src/execution/bridges/bridge.execute.ts:170]: 🙃

File: src/utils/parseError.ts [hunk 4]
```diff
@@ -142,14 +150,22 @@ export const parseError = async (
       )
     }
 
-    if (e.Code === LifiErrorCode.TransactionUnprepared) {
+    if (e.code === LifiErrorCode.TransactionUnprepared) {
```
Comment (mathiasmoeller) [inline src/utils/parseError.ts:153]: Are you sure that this one here isn't supposed to be `Code`? It's not one of our errors, could be `Code`

Comment (Addminus) [reply src/utils/parseError.ts:153]: yes i am pretty sure
image


And it is one of our errors

**Pull request merged**